### DISCUSSION TITLE
Release of v0.11

### DISCUSSION THREAD

**Discussion by silwol** [fmeringdal/rust-rrule#94] [General]
Are there any blockers preventing the release of version `0.11`? I have a project that should get `chrono-tz` updated to `0.8`, which is currently not possible due to `rrule` requiring `0.6`. Is there anything that needs to be implemented / updated before the new version can be released? Is help needed somewhere?

**Comment by fmeringdal** [comment]
Published version `0.11` now. Thanks again to @omid for the effort to upgrade `chrono-tz` to version `0.8`

**Reply by silwol** [reply]
Thanks a lot for the quick action!
